component/image/image.css

```diff
@@ -0,0 +1,19 @@
+img {
+    object-fit: cover;
+    max-width: 100%;
+}
```

Comment:
This can be applied to any image in entire project so it can be moved to reset.css

Reply:
ok